pkg/kv/kvserver/replica_rangefeed.go

```diff
@@ -231,8 +231,7 @@ func (r *Replica) rangeFeedWithRangeID(
 			// Assert that we still hold the raftMu when this is called to ensure
 			// that the catchUpIter reads from the current snapshot.
 			r.raftMu.AssertHeld()
-			return rangefeed.NewCatchUpIterator(r.Engine(),
-				args, RangefeedTBIEnabled.Get(&r.store.cfg.Settings.SV), iterSemRelease)
```

Comment:
Can we remove `RangefeedTBIEnabled` since we are removing this access?

Reply:
Thanks, missed that -- fixed over on #82450.